nexus/reconfigurator/planning/src/planner.rs

```diff
@@ -1227,20 +1228,27 @@ impl<'a> Planner<'a> {
             .map(|z| (z.id, z.image_source.clone()))
             .collect::<BTreeMap<_, _>>();
         for &sled_id in &sleds {
-            if !self
+            let zones_currently_updating = self
```

Comment:
I really only changed this to make visibility in tests better, but it helps when you "forget to update inventory after editing a blueprint".